ql/src/test/queries/clientpositive/pointlookup6.q

```diff
@@ -0,0 +1,17 @@
+create table r_table (
```

Comment:
Should these tests set the hive.optimize.point.lookup and hive.optimize.point.lookup.min explicitly since the default values could change.

Reply:
Changing `hive.optimize.point.lookup` to `false` changes the plan in the test output so I wouldn't add this explicitly. It would also turn off the whole point lookup optimization functionality and in general we don't want to turn off any functionality because it would lead to dead functionalities and make very hard to re-enable it after a while.

I added 
```
set hive.optimize.point.lookup.min=2
```
It makes sense in this test case since we have only two constants in the `IN` expressions.